### ISSUE TITLE
error setting enabled_cloudwatch_logs_exports: Invalid address to set: []string{"enabled_cloudwatch_logs_exports"}

### ISSUE THREAD

**Issue by martinzak-salsita** [terraform-providers/terraform-provider-aws#4922]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version

```
 Terraform v0.11.7
+ provider.aws v1.24.0
```

### Affected Resource(s)

```
data "aws_rds_cluster" "main_cluster" {
  cluster_identifier = "main-cluster"
}
```

### Terraform Configuration Files

```
data "aws_rds_cluster" "main_cluster" {
  cluster_identifier = "main-cluster"
}
```

### Debug Output

https://gist.github.com/martinzak-salsita/195f97370feeec7133b7810e424608e2

### Expected Behavior

We expect terraform to refresh state of cluster and continue with planing.

### Actual Behavior

```
Error: Error refreshing state: 1 error(s) occurred:

* data.aws_rds_cluster.main_cluster: 1 error(s) occurred:

* data.aws_rds_cluster.main_cluster: data.aws_rds_cluster.main_cluster: error setting enabled_cloudwatch_logs_exports: Invalid address to set: []string{"enabled_cloudwatch_logs_exports"}
```

### Steps to Reproduce

1. `terraform init`
2. `terraform apply`

### Important Factoids

Updated from provider from 1.23.0 to 1.24.0. Configuration works with the older version.

### References

Thank you!

**Comment by bflad** [comment]
Bug fix PR submitted: #4927

**Comment by bflad** [comment]
The fix has been merged into master and will release with version 1.25.0 of the AWS provider, likely middle of next week. Sorry for the trouble!

**Comment by bflad** [comment]
This has been released in [version 1.25.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1250-june-27-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
